iroh-net/src/netcheck/reportgen.rs

```diff
@@ -541,7 +543,14 @@ impl Actor {
                     match captive_portal_check.await {
                         Ok(Ok(found)) => Some(found),
                         Ok(Err(err)) => {
-                            warn!("check_captive_portal error: {:?}", err);
+                            let err: Result<reqwest::Error, _> = err.downcast();
+                            match err {
+                                Ok(req_err) if req_err.is_connect() => {
+                                    debug!("check_captive_portal failed: {req_err:#}");
+                                }
+                                Ok(req_err) => warn!("check_captive_portal error: {req_err:#}"),
+                                Err(any_err) => warn!("check_captive_portal error: {any_err:#}"),
+                            }
```

Comment:
same with these, while useful yo use I don't think they mean much for end users

Reply:
Here I jumped through some hoops to reduce the expected errors to debug but leave unexpected errors as errors.  It's hard to capture everything and maybe there's still a bit too much on WARN, but I'd like real failures to still show up.